drivers/SmartThings/zigbee-button/src/zigbee-multi-button/ikea/TRADFRI_remote_control.lua

```diff
@@ -57,6 +57,7 @@ local function added_handler(self, device)
       device:emit_component_event(comp, capabilities.button.numberOfButtons({value = 1}, {visibility = { displayed = false }}))
     end
   end
+  device:send(PowerConfiguration.attributes.BatteryPercentageRemaining:read(device))
   device:send(PowerConfiguration.attributes.BatteryVoltage:read(device))
```

Comment:
I think you probably just want to delete the `added` method here and use the parent's 

Looks like this device uses percentage, not voltage, but I missed it when fixing a long time ago: https://github.com/SmartThingsCommunity/SmartThingsEdgeDriversBeta/pull/589/files

Reply:
@greens Looking at this `added_handler` it seems to be emitting different values for button5 than the other buttons, which is slightly more complicated than what the parent's `added_handler` does. I could probably still use the parent handler but it would need to be edited a bit, should I jut edit the parent handler or just keep this `added` handler?

Reply:
ah, good eye. in that case you just want to switch the voltage read with a percentage read